snap/snapcraft.yaml

```diff
@@ -1,52 +1,88 @@
-name: spark-k8s # you probably want to 'snapcraft register <name>'
+name: spark-client
 base: core20
 version: '0.1'
-summary: Client side snap to submit spark jobs to kubernetes.
+summary: Client side scripts to submit Spark jobs to a cluster.
 description: |
   Spark is now available on Kubernetes and needs a client to submit and 
```

Comment:
Please rewrite as "The `spark-client` snap includes the scripts `spark-submit`, `spark-shell`, `pyspark` and other tools for managing Apache Spark jobs."

Reply:
Done.